Cargo.toml

```diff
@@ -32,27 +32,28 @@ wasmparser = "0.92.0"
 wasm-encoder = "0.18.0"
 wat = "1.0.49"
 
-wit-bindgen-core = { path = 'crates/bindgen-core', version = '0.2.0' }
-wit-bindgen-gen-guest-c = { path = 'crates/gen-guest-c', version = '0.2.0' }
-wit-bindgen-gen-guest-rust = { path = "crates/gen-guest-rust", version = "0.2.0" }
-wit-bindgen-gen-guest-teavm-java = { path = 'crates/gen-guest-teavm-java', version = '0.2.0' }
-wit-bindgen-gen-host-js = { path = 'crates/gen-host-js', version = '0.2.0' }
-wit-bindgen-gen-host-wasmtime-py = { path = 'crates/gen-host-wasmtime-py', version = '0.2.0' }
-wit-bindgen-gen-host-wasmtime-rust = { path = 'crates/gen-host-wasmtime-rust', version = '0.2.0' }
-wit-bindgen-gen-markdown = { path = 'crates/gen-markdown', version = '0.2.0' }
-wit-bindgen-gen-rust-lib = { path = 'crates/gen-rust-lib', version = '0.2.0' }
-wit-bindgen-guest-rust = { path = 'crates/guest-rust', version = '0.2.0' }
-wit-bindgen-host-wasmtime-rust = { path = 'crates/host-wasmtime-rust', version = '0.2.0' }
-wit-parser = { path = 'crates/wit-parser', version = '0.2.0' }
-wit-component = { path = 'crates/wit-component', version = '0.2.0' }
+wit-bindgen-core = { path = 'crates/bindgen-core', version = '0.3.0' }
+wit-bindgen-gen-guest-c = { path = 'crates/gen-guest-c', version = '0.3.0' }
+wit-bindgen-gen-guest-rust = { path = "crates/gen-guest-rust", version = "0.3.0" }
+wit-bindgen-gen-guest-teavm-java = { path = 'crates/gen-guest-teavm-java', version = '0.3.0' }
+wit-bindgen-gen-host-js = { path = 'crates/gen-host-js', version = '0.3.0' }
+wit-bindgen-gen-host-wasmtime-py = { path = 'crates/gen-host-wasmtime-py', version = '0.3.0' }
+wit-bindgen-gen-host-wasmtime-rust = { path = 'crates/gen-host-wasmtime-rust', version = '0.3.0' }
+wit-bindgen-gen-markdown = { path = 'crates/gen-markdown', version = '0.3.0' }
+wit-bindgen-gen-rust-lib = { path = 'crates/gen-rust-lib', version = '0.3.0' }
+wit-bindgen-guest-rust = { path = 'crates/guest-rust', version = '0.3.0' }
+wit-bindgen-host-wasmtime-rust = { path = 'crates/host-wasmtime-rust', version = '0.3.0' }
+wit-parser = { path = 'crates/wit-parser', version = '0.3.0' }
+wit-component = { path = 'crates/wit-component', version = '0.3.0' }
 
 [[bin]]
 name = "wit-bindgen"
 test = false
 
 [dependencies]
 anyhow = { workspace = true }
-clap = { workspace = true }
```

Comment:
I think the changes in the `[dependencies]` and `[build-dependencies]` sections may not be needed?

Reply:
you're right~